core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html

```diff
@@ -5,6 +5,6 @@
 </form>
 or
 <form ng-submit="addExploration()">
-  <input ng-model="newExplorationId" type="text" placeholder="New exploration ID">
-  <button type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
+  <input class="form-inline" ng-model="newExplorationId" type="text" placeholder="New exploration ID">
```

Comment:
I will defer the change to @seanlip. 

Note:  I can see "New exploration ID" (placeholder) comes from its `ng-model`. And this naming continues in respective JS files. @zvekovius Are you suggesting changing just the placeholder value ? Or renaming the ng-model reference itself (In that case it could be filed under separate issue) 
```
<input ng-model="newExplorationId" type="text" placeholder="New exploration ID">
```